src/components/Compare/CompareMain.tsx

```diff
@@ -44,7 +46,6 @@ const CompareMain = (props: {
   isModal?: Boolean;
   locationsViewable: number;
   isHomepage?: boolean;
-  currentCounty?: any;
```

Comment:
Note that `currentCounty` seemed to always be in-sync with `county` so I opted to populate it within the component (see line 78 below) instead of having the container component pass it in.  Let me know if I'm missing something.

Reply:
makes sense!